scripts/update-brew-cask.sh

```diff
@@ -3,15 +3,20 @@
 #script arguments
 wheel_files=( $1 )
 wheel_file=${wheel_files[0]}
-homebrew_tap_repo=$2
+arm_artifacts=( $2 )
+arm_artifact=${arm_artifacts[0]}
+intel_artifacts=( $3 )
+intel_artifact=${intel_artifacts[0]}
```

Comment:
Is there a better way to extract the artifact file name? The input is `dist/algokit*-py3-none-any.whl`